deselected_tests.yaml

```diff
@@ -725,6 +737,16 @@ gpu:
   - tests/test_common.py::test_transformers_get_feature_names_out[VotingRegressor(estimators=[('est1',Ridge(alpha=0.1)),('est2',Ridge(alpha=1))])]
   - tests/test_common.py::test_f_contiguous_array_estimator[TSNE]
   - manifold/tests/test_t_sne.py::test_tsne_works_with_pandas_output
+  - ensemble/tests/test_forest.py::test_importances[ExtraTreesClassifier-gini-float64]
```

Comment:
Could maybe add a comment for these "from ensemble out of preview" or something like that